src/utils/index.js

```diff
@@ -8,8 +8,9 @@ exports.createObject = keyArray => keyArray.reduce((acc, key) => ({ ...acc, [key
 
 exports.analyzeMatches = matches => Object.keys(matches).filter(key => removeComments(matches[key]).length);
 
-exports.calculatePercentage = (results, total) =>
-  // eslint-disable-next-line prettier/prettier
-  this.analyzeMatches(results).length / total * percentage;
+exports.calculatePercentage = (results, total, options = { skipComments: false }) => {
```

Comment:
Are you sure about this? `options = { skipComments: false }` I don't think it will work well as a default for other objects 🤔

Reply:
I'm gonna change it, for a simple approach.